sdk/src/Services/DynamoDBv2/Custom/DynamoDBTelemetry.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using Amazon.DynamoDBv2.DataModel;
+using Amazon.Runtime.Telemetry;
+using Amazon.Runtime.Telemetry.Tracing;
+using Attributes = Amazon.Runtime.Telemetry.Attributes;
+
+namespace Amazon.DynamoDBv2
+{
+    internal static class DynamoDBTelemetry
+    {
+        private static string DynamoDBTracerScope = $"{TelemetryConstants.TelemetryScopePrefix}.DynamoDBv2";
+
+        internal static string ExtractOperationName(string className, string methodName)
+        {
+            if (methodName.EndsWith("Async", StringComparison.Ordinal))
+                methodName = methodName.Substring(0, methodName.Length - 5);
```

Comment:
just curious - is this a pattern we are following to remove Async from the method name?

Reply:
We've used this pattern while adding observability to the S3 Transfer Utility HLL - https://github.com/aws/aws-sdk-net/blob/3eca86cb8c585ff490f216d29c354deec8d2a33a/sdk/src/Services/S3/Custom/Transfer/TransferUtility.cs#L552